### PR TITLE
Add auth_token to piwik metrics read

### PR DESCRIPTION
**Pull request by petzlux**
Use case when Piwik stats are not public to enable metrics collection jobs to work.

### CODE REVIEW

File: udata_piwik/client.py [hunk 1]
```diff
@@ -19,6 +19,8 @@ def analyze(method, **kwargs):
         'idSite': current_app.config['PIWIK_ID'],
         'method': method,
         'format': kwargs.pop('format', 'json'),
+        'token_auth': current_app.config['PIWIK_AUTH']
```
Comment (noirbizarre) [inline udata_piwik/client.py]: Maybe test if `current_app.config['PIWIK_AUTH']` is set and then optionnaly add it.

File: udata_piwik/client.py [hunk 2]
```diff
@@ -19,6 +19,8 @@ def analyze(method, **kwargs):
...
+
```
Comment (noirbizarre) [inline udata_piwik/client.py]: No blank line

File: udata_piwik/client.py (hunk 1 above)
Comment (petzlux) [inline udata_piwik/client.py]: You dont test presence of token in ```track``` method either ?

Comment (noirbizarre) [inline udata_piwik/client.py]: Because the `track` always require an authentication on the piwik side.

**Comment by petzlux** [comment]
@noirbizarre Okay now? Added test if toke is set

**Pull request merged**